test_docformatter.py

```diff
@@ -822,6 +822,25 @@ def test_format_code_with_syntax_error(self):
         self.assertEqual('"""\n',
                          docformatter.format_code('"""\n'))
 
+    def test_format_code_dominant_line_ending_style_preserved(self):
```

Comment:
Please also add a separate test for '\r' line endings. See my unit tests in `autopep8`.